### PR TITLE
Avoid kerploding due to Affine2x3's that are outside Fixed range.

### PR DESCRIPTION
**Pull request by rsheeter**
Short-term fix to prevent crash. Hoping to later improve the way we go about building glyphs to reduce frequency.

### CODE REVIEW

File: src/nanoemoji/write_font.py [hunk 1]
```diff
@@ -260,6 +264,7 @@ def try_reuse(self, path: str) -> Optional[ReuseResult]:
         assert path.startswith("M"), f"{path} doesn't look like a path"
 
         if self._config.reuse_tolerance == -1:
+            print("reuse disabled")
```
Comment (anthrotype) [inline src/nanoemoji/write_font.py:267]: stray print

**Review by anthrotype — APPROVED** [review]

File: src/nanoemoji/write_font.py (hunk 1 above)
Comment (anthrotype) [reply src/nanoemoji/write_font.py]: I deleted the stray print while rebasing off main branch

**Comment by anthrotype** [comment]
@googlebot I consent.

**Comment by rsheeter** [comment]
I think I need a test that just scans the code for prints!

**Pull request merged**